aiida/orm/importexport.py

```diff
@@ -405,58 +405,59 @@ def merge_extras(old_extras, new_extras, mode):
 
     old_keys = set(old_extras.keys())
     new_keys = set(new_extras.keys())
-    
+
     collided_keys = old_keys.intersection(new_keys)
     old_keys_only = old_keys.difference(collided_keys)
     new_keys_only = new_keys.difference(collided_keys)
-    
+
     final_extras = {}
-    
+
     # Fast implementations for the common operations:
     if mode == 'ncu': # 'mirror' operation: remove old extras, put only the new ones
         return new_extras
-    
-    elif mode == 'knl': # 'none': keep old extras, do not add imported ones
+
+    if mode == 'knl': # 'none': keep old extras, do not add imported ones
         return old_extras
-    
-    elif mode == 'kcu': # 'update_existing' operation: if an extra already exists,
-                               # overwrite its new value with a new one
+
+    if mode == 'kcu': # 'update_existing' operation: if an extra already exists,
```

Comment:
because it returns in the previous two if's

Reply:
I would use if at the first comparison and elif/else in the subsequent comparisons (since, it doesn't make the code more complex and you operate on the same variable)
But this is something minor and I suppose personal preference, so leave it as it is